components/common/hocs/cloud/with_get_cloud_subscription.tsx

```diff
@@ -22,12 +22,14 @@ interface UsedHocProps {
 function withGetCloudSubscription<P>(WrappedComponent: ComponentType<P>): ComponentType<any> {
     return class extends React.Component<P & UsedHocProps> {
         async componentDidMount() {
-            const {subscription, actions: {getCloudSubscription}, isCloud, userIsAdmin} = this.props;
+            // if not is cloud, not even try to destructure values from props, just return
+            if (!this.props.isCloud) {
+                return;
+            }
+            const {subscription, actions, isCloud, userIsAdmin} = this.props;
 
-            if (isEmpty(subscription) && isCloud && userIsAdmin) {
-                if (getCloudSubscription) {
-                    await getCloudSubscription();
-                }
+            if (isEmpty(subscription) && isCloud && userIsAdmin && actions?.getCloudSubscription) {
```

Comment:
The `isCloud` here is not necessary anymore

Reply:
+1 on this

Reply:
done!